app/services/analytics_events.rb

```diff
@@ -11,6 +11,20 @@
 #                 ||     ||
 
 module AnalyticsEvents
+  # @param [Boolean] success Check whether threatmetrix succeeded properly.
+  # @param [String] transaction_id Vendor-specific transaction ID for the request.
+  def account_creation_tmx_result(
+    success:,
+    transaction_id:, **extra
```

Comment:
`DdpResult#to_h` includes a lot more values than this, should we include them here?

https://github.com/18F/identity-idp/blob/b256afa4bd27eb7673d8042a8335d060eb295095/app/services/proofing/ddp_result.rb#L70-L83